vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -242,10 +264,12 @@ public CreationReturnValue createVideoBlock(String slideId, byte[] video, Long s
         return returnValue;
     }
 
-    private IVSVideo storeVideo(byte[] video, Long size, String fileName, String url, String title)
+    public IVSVideo storeVideo(byte[] video, Long size, String fileName, String url, String title)
```

Comment:
is this method ever called outside this class (or is it supposed to be)? if not, it should not be public